test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -451,16 +452,21 @@ class test_uct_cm_sockaddr : public uct_test {
         test_uct_cm_sockaddr *self;
 
         self = reinterpret_cast<test_uct_cm_sockaddr *>(arg);
+
+        if (self->m_server_start_disconnect) {
+            self->m_client->disconnect(ep);
```

Comment:
can we rename `m_client` to `m_peer`? otherwise this line contradicts with line before. Or I missed something?

Reply:
if the server is the one that initiated the disconnect, then the client needs to call uct_ep_disconnect as well.

Reply:
then let's replace it with some "local" state value which indicates was disconnect cb called on client side before or not.

Reply:
You mean like with m_cm_state?
Because if there are multiple clients, like there are in the stress tests, it won't work

Reply:
Yes but it should be per EP... If this is overcomplicated, let's rename the flag and/or add a comment. Current code looks very confusing.